charts/sourcegraph/templates/frontend/sourcegraph-frontend.Deployment.yaml

```diff
@@ -45,6 +45,26 @@ spec:
         app: sourcegraph-frontend
         deploy: sourcegraph
     spec:
+      initContainers:
+      {{- if .Values.migrator.enabled }}
+      - name: migrator
+        image: {{ include "sourcegraph.image" (list . "migrator") }}
+        args:
+        - up
+        env:
+        {{- range $name, $item := .Values.frontend.env }}
+        - name: {{ $name }}
+          {{- $item | toYaml | nindent 10 }}
+        {{- end }}
+        {{- range $name, $item := .Values.migrator.env }}
+        - name: {{ $name }}
+          {{- $item | toYaml | nindent 10 }}
+        {{- end }}
+        resources:
+          {{- toYaml .Values.migrator.resources | nindent 10 }}
+        securityContext:
+          {{- toYaml .Values.migrator.podSecurityContext | nindent 10 }}
```

Comment:
I feel like the way we use `podSecurityContext` and `securityContext` values is wrong

`podSecurityContext` should be the pod-level security context

`securityContext` (or `containerSecurityContext`) is the container-leverl security context.

That said, I still following the current convention. If we do decide to change this, it should be work on from a separate PR.

Reply:
When I set the chart up I tried to follow the existing convention from the kustomize manifests, but I agree - the distinction is not very clear.